src/components/DesignTheme/Blocks/RSS/RssBody.jsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+// import { useDispatch, useSelector } from 'react-redux';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import Parser from 'rss-parser';
+import moment from 'moment';
+import {
+  Card,
+  CardBody,
+  CardTitle,
+  CardText,
+  CardCategory,
+  CardReadMore,
+} from 'design-react-kit/dist/design-react-kit';
+
+const RssBody = ({ data, properties, intl, path, isEditMode }) => {
+  const [feedItems, setFeedItems] = useState([]);
+  moment.locale('it');
```

Comment:
Questa cosa potrebbe essere riduttiva, dato che non è solo in italiano il sito. In aggiunta, potrebbe non essere il posto giusto per farlo, se ho capito bene è un'impostazione globale. Comunque Nicola e Giulia stanno indagando sul momento giusto per impostare la lingua per risolvere alcune rogne base di Volto sui widget per le date e la ricorrenza, quindi per ora ok e rimandiamo la decisione a quando avremo capito meglio come fare.

Reply:
Si, corretto. L'ho committato per sbaglio: volevo vedere se faceva quello che mi attendevo impostando la lingua